src/js/utils/ErrorMessageUtil.js

```diff
@@ -0,0 +1,41 @@
+const MESSAGE_VARIABLE = /\{\{([^}]+)\}\}/g;
```

Comment:
The problem with the pattern is that it catch malformed variable names like `{{{foo}}}` or `{{bar {foo}}` thus I would change the character group to not match `{` as well as `}`.